.github/workflows/tests.yml

```diff
@@ -129,11 +129,13 @@ jobs:
               pytest --pyargs pywt
               python ../pywt/tests/test_doc.py
           elif [ "${REFGUIDE_CHECK}" == "1" ]; then
-              # Run doctests and check if the refguide contains entries from __all__
-              python util/refguide_check.py --doctests
+              # doctest docstrings
+              pytest --doctest-modules --pyargs pywt -v  --doctest-collect=api
               # Run Sphinx HTML docs builder, converting warnings to errors
-              pip install -r util/readthedocs/requirements.txt
-              sphinx-build -b html -W --keep-going -d _build/doctrees . doc/source doc/build
+              cd ..
+            # XXX sphinx build is broken on CI
```

Comment:
What is happening here? Debug left-over, or is something actually broken right now? If so, I think three lines can be safely deleted. The separate CI job to build the docs with the ReadTheDocs integration is green.

Reply:
Indeed, this `sphinx-build` incantation did not previously run on CI because `REFGUIDE-CHECK: [0]`, and was always broken. There's a standing offer to fix this in a follow-up though :-). https://github.com/PyWavelets/pywt/pull/747#issuecomment-2153849371

Reply:
Okay thanks, let's do that - offer is appreciated:)